addons/web/static/tests/_framework/condition_tree_editor_test_helpers.js

```diff
@@ -52,6 +52,8 @@ export const SELECTORS = {
     toggleArchive: ".form-switch",
     complexCondition: ".o_tree_editor_complex_condition",
     complexConditionInput: ".o_tree_editor_complex_condition input",
+    debugArea: ".o_domain_selector_debug_container textarea",
```

Comment:
these should not be here.
See
export const SELECTORS = {
    ...treeEditorSELECTORS,
    debugArea: ".o_domain_selector_debug_container textarea",
    resetButton: ".o_domain_selector_row > button",
};
in legacy file domain_selector_tests.js